src/pages/organize/[orgId]/people/duplicates/index.tsx

```diff
@@ -13,11 +19,42 @@ export const getServerSideProps: GetServerSideProps = scaffold(async () => {
 
 const DuplicatesPage: PageWithLayout = () => {
   const onServer = useServerSide();
+  const { orgId } = useNumericRouteParams();
+  const list = useDuplicates(orgId).data ?? [];
```

Comment:
Another consequence of the `dismissed ` change is that we should filter here the duplicates that have the value of `dismissed: false or null` to list them, and filter out the ones that have` dismissed: true`. Let me know if you want to do that filter together or you wanna figure it out for yourself :)

Reply:
As mentioned in my other comment I think this should still be a date, but filtering using truthy/falsy as the condition should work fine.